01-IAM/README.md

```diff
@@ -1,64 +1,70 @@
 
 # Cloud Foundation Toolkit Lab - 01-IAM
+[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https%3A%2F%2Fgithub.com%2Fterraform-google-modules%2Fcloud-foundation-training&cloudshell_open_in_editor=main.tf&cloudshell_tutorial=README.md&cloudshell_working_dir=01-IAM)
 
 ## Prerequisite
 
-Complete previous labs
-* **[00-Setup](../00-Setup/README.md)**
+Complete previous labs:
 
-You should have [Google Cloud SDK](https://cloud.google.com/sdk/docs/downloads-interactive) installed and configured before continuing, otherwise refer to [00-Setup](../00-Setup/README.md)
-
-## What You'll Learn
+* [00-Setup](../00-Setup/README.md)
 
-* [Cloud Foundation Toolkit](https://cloud.google.com/foundation-toolkit/)
-* [terraform-google-iam](https://github.com/terraform-google-modules/terraform-google-iam)
-* [IAM](https://cloud.google.com/iam/docs/overview)
+You should have [Google Cloud SDK](https://cloud.google.com/sdk/docs/downloads-interactive) installed and configured before continuing, otherwise refer to [00-Setup](../00-Setup/README.md)
 
 ## Identity and Access Management
 
 This lab helps you create IAM role bindings using Cloud Foundation Toolkit.
 
 At the end of the exercise, you will have user(s) in your project that have been granted access to resources within GCP.
 
+### What You'll Learn
+
+* [Cloud Foundation Toolkit](https://cloud.google.com/foundation-toolkit/)
+* [terraform-google-iam](https://github.com/terraform-google-modules/terraform-google-iam)
+* [IAM](https://cloud.google.com/iam/docs/overview)
+
 ## Task 1. Create IAM Role Bindings
 
-Use the Cloud Foundation Toolkit [IAM module](https://github.com/terraform-google-modules/terraform-google-iam) in `main.tf` to setup role bindings for a *user* and a *group*.
+Use the Cloud Foundation Toolkit [IAM module](https://github.com/terraform-google-modules/terraform-google-iam) in <walkthrough-editor-open-file filePath="main.tf">
+main.tf</walkthrough-editor-open-file> to setup role bindings for a *user* and a *group*.
```

Comment:
- Clicking on `main.tf` in Tutorial pane doesn't open the file in main editor window.
- Should open all the `tf` files together in the Open in Cloud Shell link
- nit: Is a new line necessary?

Reply:
- Looks like `walkthrough-editor-open-file` might be broken in general - it doesn't work in the example cloudshell walkthrough either. I'll remove those to avoid confusion.
- Open in Cloud Shell opens up `main.tf` by default, but all of the files are available in the UI
- Will remove newline